### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by annoyingpr**
## Briefly describe the feature or bug that this PR addresses.

Some of the dependencies and submodules required to properly run the ETL pipeline were missing in the README, small PR to update for what's needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,89 +42,95 @@ DATABASE_PASSWORD=___
...
 1. Download and install Postgres.
 
-    From the source: [https://www.postgresql.org/download/](https://www.postgresql.org/download/)
+   From the source: [https://www.postgresql.org/download/](https://www.postgresql.org/download/)
 
-    Or, on a Mac: `brew install postgresql`
+   Or, on a Mac: `brew install postgresql`
 
-2. Download and install pgAdmin 4, a management tool for Postgres. 
+2. Download and install pgAdmin 4, a management tool for Postgres.
```
Comment (atatmaja) [inline README.md:61]: We were also able to brew install pgadmin4 right? We should add that as an option here

File: README.md [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
+
 Server side code for the International Immunity Tracker
 
 # Set up
 
+## Cloning
+
+This repo contains a submodule: the shared_scripts submodule, which contains scripts used accross the SeroTracker project, in both research code and backend. This web page provides a great explanation of submodules, on which the below is based.
+
+To clone this repo, git clone will fetch the reference to the submodule but not load the submodule itself. Instead, use git clone --recurse-submodules {url}. Similarly, instead of git pull - which will fetch remote updates in the parent repo as to which submodules are required, but not insert them - use git pull --recurse-submodules.
```
Comment (atatmaja) [inline README.md:11]: let's put `` around all the terminal commands to increase visibility

File: README.md [hunk 3]
```diff
@@ -122,16 +139,19 @@ Run the script `python app/database_etl/etl_main.py`.
 Confirm that the data has indeed been migrated by checking pgAdmin 4.
 
 ## Generating Airtable Fields Config from SOP
```
Comment (atatmaja) [inline README.md:141]: @simonarocco this is no longer relevant right? should we delete

**Review by atatmaja — COMMENTED** [review]
Thanks for doing this Prannoy! Few super nitpicky comments.

File: README.md (hunk 1 above)
Comment (annoyingpr) [reply README.md:61]: Great catch, will add

**Review by atatmaja — APPROVED** [review]
LGTM

File: README.md (hunk 3 above)
Comment (simonarocco) [reply README.md:143]: ya @annoyingpr you can delete this section about generating airtable fields config from SOP

**Review by simonarocco — APPROVED** [review]

**Pull request merged**